pkg/http/rest/api.go

```diff
@@ -6,161 +6,129 @@ import (
 	"errors"
 	"fmt"
 	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/configuration"
-	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/domain"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/log"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/status"
 	"net/http"
 	"strings"
 )
 
-func ApiCall(ctx configuration.Context) (bool, error) {
-	if ctx.Metadata.ResourceConfiguration.Flags.ApplicationFlags.ZeroIssue {
-		return noIssueApiCall(ctx)
-	} else if ctx.Metadata.ResourceConfiguration.Flags.ApplicationFlags.SingleIssue {
-		return singleApiCall(ctx, ctx.IssueIds[0])
-	} else if ctx.Metadata.ResourceConfiguration.Flags.ApplicationFlags.MultipleIssue {
-		return multipleApiCall(ctx)
-	} else {
-		// TODO script?
-	}
+type CreateBodyFN func() []byte
+type GetEndpointFN func(string) string
+type JsonObjectFN func() interface{}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type JsonObjectFN should have comment or be unexported](https://app.codacy.com/app/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4570750)